src/mousetracker.js

```diff
@@ -3248,5 +3264,33 @@
             } );
         }
     }
+    
+    /**
+     * True if inside an iframe, otherwise false.
```

Comment:
I'd prefer to not have private members and methods show up in the online documentation at all. Please convert this to a `//` comment, keeping the description and the fact that it's private; that keeps the info available for folks who read the code, but it won't go in the online version.